website/docs/jellyfish/api/masternode.md

```diff
@@ -0,0 +1,127 @@
+---
+id: masternode
+title: Masternode API
+sidebar_label: Masternode API
+API slug: /jellyfish/api/masternode
+---
+
+```js
+import {Client} from '@defichain/jellyfish'
+
+const client = new Client()
+// Using client.account.
+const something = await client.masternode.method()
+```
+
+## createMasternode
+
+Creates a masternode creation transaction with given owner and operator addresses.
+
+```ts title="client.masternode.createMasternode()"
+interface masternode {
+  createMasternode (
+    ownerAddress: string,
+    operatorAddress?: string,
+    options: CreateMasternodeOptions = { utxos: [] }
+  ): Promise<string>
+}
+
+interface UTXO {
+  txid?: string
+  vout?: number
+}
+
+interface CreateMasternodeOptions {
+  inputs?: UTXO[]
+}
+```
+
+## listMasternodes
+
+Returns information about multiple masternodes.
+
+```ts title="client.masternode.listMasternodes()"
+interface masternode {
+  listMasternodes (
+    pagination: MasternodePagination = {
+      including_start: true,
+      limit: 10000
+    },
+    verbose: boolean = true
+  ): Promise<MasternodeResult>
+}
+
+enum MasternodeState {
+  PRE_ENABLED = 'PRE_ENABLED',
+  ENABLED = 'ENABLED',
+  PRE_RESIGNED = 'PRE_RESIGNED',
+  RESIGNED = 'RESIGNED',
+  PRE_BANNED = 'PRE_BANNED',
+  BANNED = 'BANNED',
+  UNKNOWN = 'UNKNOWN'
+}
+
+interface MasternodePagination {
+  start?: string
+  including_start?: boolean
+  limit?: number
+}
+
+interface MasternodeInfo {
+  ownerAuthAddress: string,
```

Comment:
Remove commas

Reply:
Remove commas. Like what you did on line 109-122

Reply:
@jingyi2811, it is done.